cypress/integration/onboarding/multi_team/existing_channel_name_spec.js

```diff
@@ -14,10 +14,15 @@
 * @param {String} channelTypeID - ID of public or private channel to create
 * @param {String} newChannelName - New channel name to assign
 */
-function verifyExistingChannelError(channelTypeID, newChannelName) {
+function verifyExistingChannelError(newChannelName, makePrivate = false) {
+    const channelTypeID = makePrivate ? '#createPrivateChannel' : '#createPublicChannel';
+
     // Click on '+' button for Public or Private Channel
     cy.get(channelTypeID).click();
 
+    // eslint-disable-next-line no-unused-expressions
+    makePrivate ? cy.get('#private').check({force: true}).should('be.checked') : cy.get('#public').should('be.checked');
```

Comment:
nit: I don't see elsewhere we're doing like this if not assigned into a variable. It could've been my preference but I suggest to make a long form of `if-else`.

Reply:
Actually, we don't need validation here, so I prefer this way:

```suggestion
    if (makePrivate) {
        cy.get('#private').check({force: true})
    }
```

Reply:
I mainly did this to ensure that whichever one was actually checked before we continued. I was unsure of if this modal had any sorts of configuration around it to maybe default it to Public or Private or maybe if a previous test opened the modal and somehow retained a particular state. My fear is that with out the check `cy.get('#public').should('be.checked')`, if we call this function with `makePrivate = false`, and SOMEHOW the the modal appeared with private checked, we would then create a channel that is private instead of a public one, which would then mess up the rest of the test.

@saturninoabril I can make it long form though, not a problem. Like...

```javascript
if (makePrivate) {
    cy.get('#private').check({force: true}).should('be.checked')
} else {
    cy.get('#public').should('be.checked')
}
```

@cometkim I'm on the fence of omitting validation. We sorta get it for "free" and feel like "why not?", just as a safety precaution? We could probably have a longer conversation around this, but the more I've been using the cypress, the more I've been wanting to throw on various assertions.

Let me know if you guys disagree completely, and I can change it.